### DISCUSSION TITLE
smb links

### DISCUSSION THREAD

**Discussion by MaFarine** [requarks/wiki#6671] [Help / Questions]
Hi there,

I am looking for the setting that allows to click on smb links.

I have a dev wiki and a prod wiki and on the dev one I can open them but not on the other..

Could you help me with that?

Thanks!

OS: Linux - Debian (bullseye) 11 x64
Wiki.js version: 2.5.274
Database engine: 10.5.15-MariaDB-0+deb11u1

**Comment by MaFarine** [comment]
Hello,

I reopen this discussion because our IT made a tweak (I don't know exactly what they did) to make smb links work but then we discovered that clicking on toc items wouldn't redirect to the section anymore.

They ask me to choose between the functionalities but want them both.

Any ideas on how it could be achieved?

Thanks a lot and best regards,
Mathieu

**Reply by NGPixel** [reply]
This is more of a browser security limitation.

Open the dev tools and see if you get an error when clicking on a link / toc item?

**Reply by MaFarine** [reply]
Mmh so yes it sends me an error but not on all pages.
> TypeError: Target must be a Number/Selector/HTMLElement/VueComponent, received undefined instead.

So I guessed it was not a browser problem.

**Reply by NGPixel** [reply]
Are you using links inside your headers?

**Reply by MaFarine** [reply]
No no they are only in paragraphs / lists
![image](https://github.com/requarks/wiki/assets/82103273/4f45bb8a-40eb-4a13-9db5-c1c7188ee6bb)

**Comment by MaFarine** [comment]
So just so you know.

We updated Wiki and it works again so I'll close the topic!

Thanks for your help!
